tests/chainer_tests/test_link.py

```diff
@@ -2466,4 +2466,43 @@ def test_to_intel64(self):
         assert l.child.to_method_called == 1
 
 
+class TestSerialize(unittest.TestCase):
+    def setUp(self):
+        self.array = numpy.array([1, 2, 3], dtype=numpy.float32)
+        self.serializer = mock.MagicMock(return_value=self.array)
+
+        # Create an empty link to populate.
```

Comment:
Is this comment appropriate? `link` is not empty a few lines below.

Reply:
Removed the comment.